### ISSUE TITLE
Fix generation of url query parameter's for 'filters' parameter in 'get_milestones' method (and other methods with filtering)

### ISSUE THREAD

**Issue by l0kifs** [qase-tms/qase-python#98]
There is an [article](https://github.com/qase-tms/qase-python/blob/master/qaseio/docs/MilestonesApi.md#get_milestones) in qase-python documentation regarding the `get_milestones` metod with `filters` parameter.

If I use it as is, I'm receiving an 400 HTTP error:
```
Traceback (most recent call last):
  File "C:\Users\Sergei\PycharmProjects\sandbox\qaseio_test.py", line 73, in <module>
    get_milestones()
  File "C:\Users\Sergei\PycharmProjects\sandbox\qaseio_test.py", line 40, in get_milestones
    milestone = milestones_api.get_milestones(project_code, filters=filters)
  File "C:\Users\Sergei\PycharmProjects\sandbox\venv\lib\site-packages\qaseio\api\milestones_api.py", line 729, in get_milestones
    return self.get_milestones_endpoint.call_with_http_info(**kwargs)
  File "C:\Users\Sergei\PycharmProjects\sandbox\venv\lib\site-packages\qaseio\api_client.py", line 884, in call_with_http_info
    return self.api_client.call_api(
  File "C:\Users\Sergei\PycharmProjects\sandbox\venv\lib\site-packages\qaseio\api_client.py", line 422, in call_api
    return self.__call_api(resource_path, method,
  File "C:\Users\Sergei\PycharmProjects\sandbox\venv\lib\site-packages\qaseio\api_client.py", line 206, in __call_api
    raise e
  File "C:\Users\Sergei\PycharmProjects\sandbox\venv\lib\site-packages\qaseio\api_client.py", line 199, in __call_api
    response_data = self.request(
  File "C:\Users\Sergei\PycharmProjects\sandbox\venv\lib\site-packages\qaseio\api_client.py", line 448, in request
    return self.rest_client.GET(url,
  File "C:\Users\Sergei\PycharmProjects\sandbox\venv\lib\site-packages\qaseio\rest.py", line 236, in GET
    return self.request("GET", url,
  File "C:\Users\Sergei\PycharmProjects\sandbox\venv\lib\site-packages\qaseio\rest.py", line 230, in request
    raise ApiException(http_resp=r)
qaseio.exceptions.ApiException: Status Code: 400
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Date': 'Fri, 10 Mar 2023 10:57:29 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Cache-Control': 'no-cache, private', 'X-RateLimit-Limit': '600', 'X-RateLimit-Remaining': '584', 'Strict-Transport-Security': 'max-age=15724800; includeSubDomains'})
HTTP response body: {"status":false,"errorMessage":"Data is invalid.","errorFields":[{"field":"filters","error":"The filters must be an array."}]}
```

Cause is a bug in url query parameter generation here:
https://github.com/qase-tms/qase-python/blob/88ad2c0533355902c50cf43a96a487d637f08128/qaseio/qaseio/api_client.py#L791

To fix it, you can replace this line with
```python
if param_location == 'query' and base_name == 'filters':
    for param_value_full in param_value.to_dict().items():
        params[param_location].append(param_value_full)
else:
    param_value_full = (base_name, param_value)
    params[param_location].append(param_value_full)
```

**Comment by n3r** [comment]
Hello @l0kifs!

This issue should have been fixed in #101. Can you check, please?

**Comment by l0kifs** [comment]
Filtering works fine now. Thanks a lot!
